subprojects/gradle/code-ownership/src/main/kotlin/com/avito/android/CheckProjectDependenciesOwnershipTask.kt

```diff
@@ -58,6 +59,13 @@ abstract class CheckProjectDependenciesOwnershipTask : DefaultTask() {
         }
     }
 
+    private fun projectDependencies() = project.internalModule
```

Comment:
How will it behave? If we have changes in the platform-module will it be ignored?

Reply:
This is the ownership plugin. It checks project's dependencies.
In this change, the plugin will ignore owner of platform module. It has no code and reused everywhere.